signingscript/src/signingscript/sign.py

```diff
@@ -1551,13 +1554,20 @@ def verify_mar_signature(cert_type, fmt, mar, keyid=None):
     else:
         crosscert = None
 
+    if comment and orig_path.endswith('.msi'):
+        log.info(f"Using comment '{comment}' to sign {orig_path}")
+    elif comment:
+        log.info("Not using specified comment to sign {orig_path}, not yet implemented.")
```

Comment:
Guessing you were intending to use an f-string here?

Reply:
Good catch, yes.